### DISCUSSION TITLE
Ability to rearrange the order of current tab

### DISCUSSION THREAD

**Discussion by KorigamiK** [zen-browser/desktop#2423] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

To be honest this is a feature request. I am looking for a keybind that can allow me to move the current tab up or down the list.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

Mozilla Zen 1.0.1-a.2

### What platform are you seeing the problem on?

Linux

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1636/ability-to-rearrange-the-order-of-current-tab">ZEN-1636 Ability to rearrange the order of current tab</a></p>

**Comment by JSGRD22** [comment]
Ctrl+Shift+PgUp/PgDn is the standard keybind for browsers

**Comment by KorigamiK** [comment]
> Ctrl+Shift+PgUp/PgDn is the standard keybind for browsers

Thanks, but is there any way that I can remap this keybind to `Alt-I/U`?

**Comment by JSGRD22** [comment]
> Thanks, but is there any way that I can remap this keybind to `Alt-I/U`?

Doesn't seem possible with the key bindings listed in settings. Not sure if there's something else that can be modified to enable this

**Comment by RayZ3R0** [comment]
Moving this to discussions as its a suggestion.
